src/tsickle.ts

```diff
@@ -456,6 +496,15 @@ class Annotator extends ClosureRewriter {
       return true;
     }
 
+    if (hasExportingDecorator(node, this.program.getTypeChecker())) {
+      let {name} = node as (
+          ts.ClassDeclaration | ts.MethodDeclaration | ts.PropertyDeclaration |
+          ts.AccessorDeclaration);
+      if (name) {
```

Comment:
This is weird.  The "as" expression is a coercion, but we can't guarantee that node is one of these types.  Maybe you meant to check node.kind here?